script/generate_async.py

```diff
@@ -99,7 +143,29 @@
 
         top_level_content_patch.append(line)
 
-    with open(f"{BASE_PATH}/grafana_client/elements/_async/__init__.py", "w") as fp:
+    with open(f"{target}/__init__.py", "w") as fp:
         fp.write("\n".join(top_level_content_patch) + "\n")
 
-    subprocess.call(["poe", "format"])
+    subprocess.call(["black", target])
+    subprocess.call(["isort", target])
+
+    exitcode = 0
+
+    if run_check:
+        command = f"diff -u {TARGET} {target}"
```

Comment:
```suggestion
        command = f"diff -x __pycache__ -u {TARGET} {target}"
```

Found the guilty part, pycache!